### PR TITLE
Isolated Network should be false by default

### LINKED ISSUE #515: DeployResourcesWithIsolatedNetwork is missing default value.
#470 has added some new params into the bicep file.

- [ ] `param DeployResourcesWithIsolatedNetwork bool` does not have a default value which breaks `azd down` - Suggestion - `false`
- [ ] /workspaces/rag-experiment-accelerator/infra/main.bicep - From line 93 - 106 new values have been added and they start with a capital letter and the rest of the file doesn't

### PR DESCRIPTION
**Pull request by ross-p-smith**
Fixes #515 

When cloning fresh and using `azd up`, the deployment no longer works. This is because a boolean flag does not have a default value. This value was used to switch isolated networking on/off. We could do the same thing by using the presence of a vnet address space

This is just a suggestion.

I've also kept the variable names in the same style.

### CODE REVIEW

**Review by jedheaj314 — APPROVED** [review]
LGTM

**Comment by martinpeck** [comment]
are *all* of these changes part of fixing this bug?

**Comment by ross-p-smith** [comment]
> are _all_ of these changes part of fixing this bug?

Previous update of the bicep file did not re-generate the json file. I suggest we add something to the pre-commit hook to generate the json from the bicep

**Comment by ross-p-smith** [comment]
@martinpeck - Added #519 so that this doesnt happen again

**Comment by martinpeck** [comment]
Yep. Got it. Thanks!

File: README.md [hunk 1]
```diff
@@ -163,25 +163,23 @@ az deployment sub create --subscription <subscription-id> --location <location>
 
 Or
 
-To deploy with isolated network use following command. (Replace the parameter values where needed)
+To deploy with isolated network use following command. (Replace the parameter values where needed). If you do not fill in all three, the deployment will not deploy this to an isolated network.
```
Comment (martinpeck) [inline README.md:166]:
```suggestion
To deploy with isolated network use following command. Replace the parameter values with the specifics of your isolated network. You **must** supply all three parameters (i.e. `vnetAddressSpace`, `proxySubnetAddressSpace` and `subnetAddressSpace`) if you wish to deploy to an isolated network.
```

**Review by martinpeck — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ross-p-smith) [reply README.md:166]: have added in #520

**Pull request merged**